client/operation_join.go

```diff
@@ -14,4 +16,12 @@ type operationJoinResponse struct {
 
 func (op operationJoinResponse) Process(state *albionState, uploader iuploader) {
 	log.Debugf("Got JoinResponse operation...")
+
+	loc, err := strconv.Atoi(op.Location)
+	if err != nil {
+		log.Debugf("Unable to convert zoneID to int. Probably an instance.. ZoneID: %v", op.Location)
+		return
+	}
+
+	state.LocationId = loc
```

Comment:
Can you add a debug line similar to this one: https://github.com/Regner/albiondata-client/blob/master/client/operation_get_game_server_by_cluster.go#L22

Reply:
done